components/IndicatorStat/IndicatorStatDataHandler.tsx

```diff
@@ -8,20 +8,44 @@ interface DataHandlerProps {
   dataLoading?: boolean;
   valueOptions?: ValueOptions;
   note?: ContentNote;
+  decimalCount?: number;
```

Comment:
The decimals affect the value, so it stands to reason that this property (`decimalCount`) should be added to ValueOptions. That'll ensure as few changes as possible.

Reply:
Moved deimalCount to valueOptions